versioned_docs/version-v2.18.x/user-guide/install-zowe-server-install-wizard copy.md

```diff
@@ -0,0 +1,80 @@
+# Troubleshooting Zowe Server Install Wizard
```

Comment:
@1000TurquoisePogs - Should this file AND `install-zowe-server-install-wizard-1-0-0.md` exist in both v2.18 and the V3 versions?

Reply:
I dont understand the origin of the "copy" files in this PR, but this file appears to be the equivalent of troubleshooting-zowe-server-install-wizard.md in v3

It is seen in this location in v3
<img width="265" alt="image" src="https://github.com/user-attachments/assets/30b773ab-4742-4064-80fa-318c9ebf483c" />

...But, the location in v3 doesnt look right either, because it's a troubleshooting document outside the troubleshooting category of the docs.

It's a page worth having, but its location in both versions seems wrong.